modules/tinymce/src/core/main/ts/file/Conversions.ts

```diff
@@ -22,18 +22,28 @@ const blobUriToBlob = (url: string): Promise<Blob> =>
     }));
 
 const extractBase64Data = (data: string): string => {
-  const matches = /([a-z0-9+\/=\s]+)/i.exec(data);
+  const whitespaceRemovedData = data.replace(/\s+/g, '');
```

Comment:
I don't think we need this, the newlines are still valid, they just can't be url encoded?

Note the `\s` marked as valid in the existing regex 🙂

Reply:
Great point! Thanks for spotting it 😄